build.gradle

```diff
@@ -158,8 +158,6 @@ distributions {
                 [
                     'configuration/vsp-with-matching.yml',
                     'configuration/vsp-with-matching-custom.yml',
-                    'configuration/vsp-no-eidas.yml',
```

Comment:
~Should the vsp without eidas config be kept? Maybe it could be renamed though?~
Looks like it's exactly the same as the `verify-service-provider.yml` config except with the `europeanIdentity` key still in so probably fine.